### PR TITLE
Feature accept-mission button changes to continue-mission button

### PR DESCRIPTION
**Pull request by Fry-Fr**
# Description

Fixes #

- What work was done?
  adding continue-the-mission className and also adding conditionals that check state changes in tasks, and when hasRead is completed this will change the accept-the-mission button too appear with a different color and also change the text to say continue the mission.
When continue button is clicked pushes the route to where child is at in current tasks, and skips the gamemode route.

- Why was this work done?
  this feature adds to the UI/UX design to make it easier to access where the progress was last updated.

- Any relevant links or images / screenshots
6aea3f730fcd47aba1c900448b01fdc6
fa21c84e990a41a8a72fa3de3a19369d

### CODE REVIEW

**Comment by Fry-Fr** [comment]
Trello card:
 [Accept the Mission - Rejoin the Mission](https://trello.com/c/sWksBMyl/808-accept-the-mission-rejoin-the-mission)

File: src/styles/less/ChildDashboard.less [hunk 1]
```diff
@@ -45,6 +47,21 @@
...
+    background-color: #f5f24b;
+
+    img {
+      z-index: 1;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+    }
+  }
+
```
Comment (felipe-sq) [inline src/styles/less/ChildDashboard.less:64]: From a UX/UI perspective, this is a much needed fix. Great work keeping the “continue mission” image consistent with the overall styling theme!

**Review by felipe-sq — COMMENTED** [review]
The code looks good at first glance but was not functioning as expected on my local machine. I will re-attempt to test the changes to the accept-mission/continue-mission button locally.

File: src/styles/less/ChildDashboard.less (hunk 1 above)
Comment (StnsGeneral) [reply src/styles/less/ChildDashboard.less:64]: I agree with this sentiment. Great job getting this started. I would also ask for a clarification on the coloring scheme. I am not a massive fan of the colors they are set to right this moment. I am a fan that they are different colors, they just look a little out of place to me. It makes sense, just might want to ask if those are the colors that they want in the long run.

**Review by StnsGeneral — DISMISSED** [review]
The ternary for starting the continue mission vs accept mission is nice. I would like to see the page that is continues on being set depending on where they are. I.e. '/gameplay/mission/draw' or '/gameplay/mission/write'. Would be quite simple to implement this logic.

**Review by derekjpeters — APPROVED** [review]

**Review by derekjpeters — APPROVED** [review]
Great job on the functionality of this.  Get one more review and work with Ash to merge.

**Pull request closed without merging**